installation/reverse-proxy.md

```diff
@@ -39,6 +39,9 @@ server {
     listen                                    80;
     server_name                               mydomain_or_myip;
 
+    # Enable web sockets, e.g. used by the log viewver in the new OH4 GUI
+    proxy_http_version                        1.1;
```

Comment:
This doesn’t need to be set here, already set in the location block below, please revert.